docs/changelog.rst

```diff
@@ -2,6 +2,27 @@
 Changelog
 *********
 
+
+1.4.0 (2022-08-22)
+------------------
```

Comment:
What about `temporarily_pull_file`?

Reply:
Good catch, I added that back into the changelog.